backend/School.js

```diff
@@ -28,26 +38,24 @@ class School {
    * @param {Student} student - Student object
    * @return {Student} Enrolled student
    */
+
   enrollStudent(className, student) {
-    // Your code here
+    student = new Student(req.body.name, req.body.age, req.body.city, req.body.grade)
```

Comment:
req.body doesnt exist here, you gotta refer to the keys in the object as the argument you passed through the function